components/SplitsDisplayTab.vue

```diff
@@ -11,14 +11,17 @@
       <loading-switch v-model="savedSettings.graphMedianAttemptHline" class="mb-2">
         Display median attempt's times as a horizontal line
       </loading-switch>
-      <loading-switch v-model="savedSettings.cumulateSplits">
+      <loading-switch v-model="savedSettings.cumulateSplits" class="mb-2">
         Show pace so far instead of individual split times
         <span v-b-tooltip.hover
               title="For big splitfiles (lots of splits or attempts), this might slow down the page for a while, be patient."
               class="help-question text-warning">
         <font-awesome-icon icon="warning" class="warning-icon"/>
       </span>
       </loading-switch>
+      <loading-switch v-model="plotByDate">
+        Plot runs by date
```

Comment:
Very small details, but so far we used "attempts" instead of "runs" in all the options' texts
```suggestion
        Plot attempts by date
```